tool/site/client/src/services/test-packs/index.ts

```diff
@@ -80,31 +90,37 @@ function createPrescriptions(
 const validFhirPrescriptionTypes = ["acute", "repeat-prescribing", "repeat-dispensing"]
 
 function createAcutePrescription(
+  prescriptionType: PrescriptionType,
   patient: fhir.BundleEntry,
   prescriber: fhir.BundleEntry,
+  places: Array<fhir.BundleEntry>,
   prescriptionRows: PrescriptionRow[],
   nominatedPharmacy: string,
   prescriptions: any[]
 ) {
-  const prescription = createPrescription(patient, prescriber, prescriptionRows)
+  const prescription = createPrescription(prescriptionType, patient, prescriber, places, prescriptionRows)
   const bundle = JSON.parse(prescription)
   updateNominatedPharmacy(bundle, nominatedPharmacy)
   prescriptions.push(JSON.stringify(bundle))
 }
 
 function createRepeatPrescribingPrescriptions(
-  prescriptionRow: PrescriptionRow,
+  prescriptionType: PrescriptionType,
   patient: fhir.BundleEntry,
   prescriber: fhir.BundleEntry,
+  places: Array<fhir.BundleEntry>,
   prescriptionRows: PrescriptionRow[],
   nominatedPharmacy: string,
   prescriptions: any[]
 ) {
+  const prescriptionRow = prescriptionRows[0]
   const repeatsAllowed = prescriptionRow.repeatsAllowed
   for (let repeatsIssued = 0; repeatsIssued <= repeatsAllowed; repeatsIssued++) {
```

Comment:
just checking the logic here, if `repeatsAllowed` is null because the spread sheet is empty this `for` block will not error it just won't run?

Reply:
realized this is existing code so fairly sure it must be working